smtp.go

```diff
@@ -20,6 +20,8 @@ type SMTP struct {
 }
 
 // CheckSMTP performs an email verification on the passed domain via SMTP
+//  - domain is the passed email domain
+//  - username is used to check the deliverability of specific email address, if server is catch-all server, username will not be checked
```

Comment:
the username is used to check the deliverability of a specific email address, if the server is a catch-all server, the username will not be checked

Reply:
split the comment into multi-lines?

Reply:
Yes, that's better.